src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAroundBasic.java

```diff
@@ -89,13 +90,13 @@ private void testCasts()
     private void testQuestions()
     {
 
-        boolean b = (1 ==2) ? false : true; //warn
+        boolean b = (1 ==2) ? false : true; // violation ''==' is not followed by whitespace.'
     }
 
     /** star test **/
     private void starTest()
     {
-        int x = 2 * 3* 4; //warn
+        int x = 2 * 3* 4; // violation ''*' is not preceded with whitespace.'
```

Comment:
Let's avoid .*

Reply:
the violation is for `3*`, in this case we need to compulsory use `'*'`

Reply:
It is not .* , let's keep it as it is , my bad